.travis.yml

```diff
@@ -1,25 +1,26 @@
 language: node_js
 node_js:
-  - "8"
+  - "10"
 services:
   - docker
 cache:
   yarn: true
   directories:
     - node_modules
 
-install:
-  # nothing
+# install current version of yarn
+before_install:
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
+  - export PATH="$HOME/.yarn/bin:$PATH"
```

Comment:
@appinteractive is this redundant?

Reply:
Yes it can be removed. Was caused by the unversionated bode image before.